static/index.html

```diff
@@ -232,6 +235,33 @@ <h1 class="page-header">Red October Management</h1>
 				</form>
 			</div>
 		</section>
+		<section class="row">
+			<div id="restore" class="col-md-6">
+				<h3>Restore</h3>
+
+				<form id="user-restore" class="ro-user-restore" role="form" action="/restore" method="post">
+					<div class="feedback restore-feedback"></div>
+
+					<div class="form-group row">
+						<div class="col-md-6">
+							<label for="restore-user">User name</label>
+							<input type="text" name="Name" class="form-control" id="restore-user" placeholder="User name" required />
+						</div>
+						<div class="col-md-6">
+							<label for="restore-user-pass">Password</label>
+							<input type="password" name="Password" class="form-control" id="restore-user-pass" placeholder="Password" required />
+						</div>
+					</div>
+					<div class="form-group row">
+						<div class="col-md-6">
+							<label for="restore-user-time">Delegation Time <small>(e.g., 2h34m)</small></label>
+							<input type="text" name="Time" class="form-control" id="restore-user-time" placeholder="1h" required />
```

Comment:
Yes, but that's a comment for a UI improvement process. All the
placeholders would need to be changed; this PR keeps the current style to
remain consistent in the scope of this effort.